.github/workflows/coverage.yml

```diff
@@ -0,0 +1,169 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Does this empty field mean it will run with every pull request?

Reply:
Yes, what do you think on extending on this by disabling the CI on draft PRs?

Reply:
I just wanted to learn what it means to make sure I understand the change.  IIRC docs don't explicitly say what is the behavior when this field is empty. :D  

I'm not sure, why wouldn't we want to run it on drafts? Draft is still in development, so it would be useful at that stage to see if tests are passing. I'd keep it.